src/main/java/com/easypost/net/EasyPostResource.java

```diff
@@ -186,17 +264,22 @@ public void merge(final EasyPostResource obj, final EasyPostResource update) {
                 } catch (Exception e) {
                     // TODO: Address situation below
                     /*
-                      The method getSmartrates() on the Shipment object is causing this exception.
-                      Since it found a method with "get" in the name, it expects there to be a "set" equivalent.
-                      There is not, causing this exception to be thrown, although nothing wrong has really happened.
-                      This code block was copy-pasted from StackOverflow: https://stackoverflow.com/a/7526414/13343799
-                      Per the comments, there are some built-in expectations for how this will work,
-                      and should eventually be re-written or removed entirely
-                      (explore returning a brand-new object rather than modifying the existing one).
-                      For now, the easiest fix would be to
-                      a) just ignore this exception, or
-                      b) rename getSmartrates() in the Shipment class to just smartrates()
-                      (similar to how the other methods are named).
+                     * The method getSmartrates() on the Shipment object is causing this exception.
+                     * Since it found a method with "get" in the name, it expects there to be a
+                     * "set" equivalent.
+                     * There is not, causing this exception to be thrown, although nothing wrong has
+                     * really happened.
+                     * This code block was copy-pasted from StackOverflow:
+                     * https://stackoverflow.com/a/7526414/13343799
+                     * Per the comments, there are some built-in expectations for how this will
+                     * work,
+                     * and should eventually be re-written or removed entirely
+                     * (explore returning a brand-new object rather than modifying the existing
+                     * one).
+                     * For now, the easiest fix would be to
+                     * a) just ignore this exception, or
+                     * b) rename getSmartrates() in the Shipment class to just smartrates()
+                     * (similar to how the other methods are named).
```

Comment:
All these extra stars aren't necessary since a `/* ... */ will comment out everything in-between.